drones/common/drone_state.py

```diff
@@ -0,0 +1,33 @@
+"""Drone state bundle
+"""
+
+from dataclasses import dataclass
+
+
+@dataclass(init=False, repr=True, eq=True, order=False, unsafe_hash=False, frozen=False)
```

Comment:
There is no need for overriding default parameters in dataclass e.g. `frozen` is set to `False` by default.